coach/src/concept-coach/index.cjsx

```diff
@@ -142,8 +142,8 @@ class ConceptCoachAPI extends EventEmitter2
       )
 
 
-  open: (props) ->
-    openProps = _.extend({}, props, open: true)
+  open: (props...) ->
+    openProps = _.extend({}, props..., open: true)
     openProps.triggeredFrom = _.pick(props, 'moduleUUID', 'collectionUUID')
```

Comment:
hm, doesn't the splat `...` operator turn the arguments into an array, which would mean that `props` is an array ?  that seems weird to pass to `_.extend`, but if that's true can we call the var something other than `props` ?  When I see that name i think it's an object